chaste_codegen/data/tests/chaste_reference_models/GRL1Opt/dynamic_matsuoka_model_2003.cpp

```diff
@@ -1539,7 +1539,7 @@
             const double var_x108 = var_x61 * var_x66;
             const double var_x110 = var_x62 * var_x79;
             const double var_x111 = var_x110 * var_x65;
-            const double var_x113 = 1.0 * var_x73;
+            const double var_x113 = var_x73;
```

Comment:
Ah, I see why the bizarre effect happens! Having the 1.0* removed just when printing means the assignment isn't elided in the Jacobian calculation.

Reply:
yes exactly, that's a downside, however I think it's an acceptable compromise